packages/host/app/services/card-service.ts

```diff
@@ -82,10 +82,7 @@ export default class CardService extends Service {
     return card;
   }
 
-  async loadModel(url: string | URL | undefined): Promise<Card | undefined> {
-    if (!url) {
-      return;
-    }
+  async loadModel(url: URL): Promise<Card> {
```

Comment:
by making this API more precise it simplifies the types for the consumers of this API